app/assets/javascripts/workers/aether_worker.js

```diff
@@ -19,8 +19,16 @@ var languagesImported = {};
 
 var ensureLanguageImported = function(language) {
   if (languagesImported[language]) return;
-  if (language === 'html') return;
-  importScripts("/javascripts/app/vendor/aether-" + language + ".js");
+  if (language === 'html' || language === 'javascript') return;
+  //Detect very modern javascript support.
+  try {
```

Comment:
Similar check to the esper engine to ensure the right plugin version chosen for the esper engine.